src/node_crypto.cc

```diff
@@ -3899,18 +3916,25 @@ void DiffieHellman::Initialize(Environment* env, Local<Object> target) {
 
   env->SetProtoMethod(t, "generateKeys", GenerateKeys);
   env->SetProtoMethod(t, "computeSecret", ComputeSecret);
-  env->SetProtoMethod(t, "getPrime", GetPrime);
-  env->SetProtoMethod(t, "getGenerator", GetGenerator);
-  env->SetProtoMethod(t, "getPublicKey", GetPublicKey);
-  env->SetProtoMethod(t, "getPrivateKey", GetPrivateKey);
+  env->SetProtoMethod(t, "getPrime", GetPrime,
+                      SideEffectType::kHasNoSideEffect);
+  env->SetProtoMethod(t, "getGenerator", GetGenerator,
+                      SideEffectType::kHasNoSideEffect);
+  env->SetProtoMethod(t, "getPublicKey", GetPublicKey,
+                      SideEffectType::kHasNoSideEffect);
+  env->SetProtoMethod(t, "getPrivateKey", GetPrivateKey,
+                      SideEffectType::kHasNoSideEffect);
   env->SetProtoMethod(t, "setPublicKey", SetPublicKey);
   env->SetProtoMethod(t, "setPrivateKey", SetPrivateKey);
 
   Local<FunctionTemplate> verify_error_getter_templ =
       FunctionTemplate::New(env->isolate(),
                             DiffieHellman::VerifyErrorGetter,
                             env->as_external(),
-                            Signature::New(env->isolate(), t));
+                            Signature::New(env->isolate(), t),
+                            // TODO(TimothyGu): should be deny
+                            ConstructorBehavior::kAllow,
```

Comment:
Should be `/* length */ 0, ConstructorBehavior::kAllow,`.  Likewise on line 3968.

Reply:
Oops, good catch. Fixed.